forge-gui/res/cardsfolder/upcoming/the_fall_of_kroog.txt

```diff
@@ -0,0 +1,8 @@
+Name:The Fall of Kroog
+ManaCost:4 R R
+Types:Sorcery
+A:SP$ ChoosePlayer | ValidTgts$ Player.Opponent | TgtPrompt$ Select target opponent | Choices$ TargetedPlayer | SubAbility$ DBDestroy | StackDescription$ SpellDescription | SpellDescription$ Choose target opponent. Destroy target land that player controls. The Fall of Kroog deals 3 damage to that player and 1 damage to each creature they control.
+SVar:DBDestroy:DB$ Destroy | ValidTgts$ Land.ControlledBy ChosenPlayer | TgtPrompt$ Select target land that player controls | SubAbility$ DBDealDamage
```

Comment:
Follow _Yosei, the Morning Star_ **very closely** for this one

Reply:
Uuh, that's not very close yet

Do the targeting the same way with fallback to RememberOriginalTargets because the rules allow the damage to happen even if the player gains hexproof in response